gctscript/modules/gct/common.go

```diff
@@ -387,6 +389,32 @@ func convertSMA(a objects.Object) ([][]string, error) {
 	return bucket, nil
 }
 
+func convertCorrelationCoefficient(a objects.Object) ([][]string, error) {
+	obj, ok := objects.ToInterface(a).(*indicators.Correlation)
+	if !ok {
+		return nil, errors.New("casting failure")
+	}
+
+	var bucket = [][]string{
+		{
+			indicators.CorrelationCoefficient,
+		},
+		{
+			fmt.Sprintf("Period:%d", obj.Period),
```

Comment:
We could pass in both objects to get the relation, but that's not stopping this merge.

Reply:
Resolving as discussed.